.github/workflows/examples-java-push.yml

```
@@ -55,3 +55,20 @@ jobs:
           name: examples-java-reports
           path: '**/build/reports/*'
           retention-days: 5
+  deploy:
+    runs-on: ubuntu-18.04
+    needs: test
+#    if: github.ref == 'refs/heads/main'
+    defaults:
+      run:
+        working-directory: examples/java
+    steps:
+      - name: Deploy to PaaS
+        uses: citizen-of-planet-earth/cf-cli-action@master
```

This should at least be version pinned, but should also include the SHA
```suggestion
        uses: citizen-of-planet-earth/cf-cli-action@v2
```